playwright/pages/SettingsPage.ts

```diff
@@ -0,0 +1,76 @@
+import { type Locator, type Page, expect } from "@playwright/test";
+
+import { GitRepositoryConnectDialog } from "./components/GitRepositoryConnectDialog";
+import { GitRepositoryDisconnectDialog } from "./components/GitRepositoryDisconnectDialog";
+import { SliceMachinePage } from "./SliceMachinePage";
+
+export class SettingsPage extends SliceMachinePage {
+  readonly breadcrumbLabel: Locator;
+  readonly installGitHubButton: Locator;
+  readonly gitRepositoryConnectDialog: GitRepositoryConnectDialog;
+  readonly gitRepositoryDisconnectDialog: GitRepositoryDisconnectDialog;
+  readonly unauthenticatedErrorTitle: Locator;
+  readonly unauthorizedErrorTitle: Locator;
+  readonly unknownErrorTitle: Locator;
+
+  constructor(page: Page) {
+    super(page);
+
+    /**
+     * Components
+     */
+
+    /**
+     * Static locators
+     */
+    this.breadcrumbLabel = this.breadcrumb.getByText("Settings", {
+      exact: true,
+    });
+    this.installGitHubButton = page.getByRole("button", {
+      name: "GitHub",
+      exact: true,
+    });
+    this.gitRepositoryConnectDialog = new GitRepositoryConnectDialog(page);
+    this.gitRepositoryDisconnectDialog = new GitRepositoryDisconnectDialog(
+      page,
+    );
+    this.unauthenticatedErrorTitle = page.getByText(
+      "It seems like you are logged out",
+      { exact: true },
+    );
+    this.unauthorizedErrorTitle = page.getByText(
+      "It seems like you do not have permission",
+      { exact: true },
+    );
+    this.unknownErrorTitle = page.getByText("Unable to fetch data", {
+      exact: true,
+    });
+  }
+
+  /**
+   * Dynamic locators
+   */
+  getConnectGitRepoButton(name: string): Locator {
+    return this.page
+      .locator("div", { hasText: name })
+      .getByRole("button", { name: "Connect", exact: true });
+  }
+
+  getDisconnectGitRepoButton(name: string): Locator {
+    return this.page
+      .locator("div", { hasText: name })
+      .getByRole("button", { name: "Disconnect", exact: true });
+  }
+
+  /**
+   * Actions
+   */
+  async goto() {
+    await this.page.goto("/settings");
+    await expect(this.breadcrumbLabel).toBeVisible();
```

Comment:
After some check, it seems it's because of the Next Error Popup grrrr

Not 100% sure but you could verify with having the screenshot.

<img width="1760" alt="Screenshot 2024-02-23 at 00 43 52" src="https://github.com/prismicio/slice-machine/assets/19946868/3d6ba474-aa77-4642-b84a-edb4b3c48b17">

Reply:
So breadcrumbLabel exist but hidden

Reply:
I would suggest if it's the modal that you just share from `SliceMachinePage` a function to close the modal manually... It's super hacky, but we should not have this modal for expected error. Then you just have to call the closeNextErrorMocal modal or something like that after loading the page.

Maybe you would also like to invest time in removing the modal but since it's Next and we don't want to keep it I would really try to keep it minimal here.

Reply:
Btw today we run Playwright with SM in dev mode so we have the latest version of the package. It could be different if we use the alpha version for example. But not done at all yet. More reason not to invest too much here I guess.

Reply:
We could build packages rather than running the dev scripts. Would that solve this problem, and technically get us more accurate testing?

Reply:
Did you check @bapmrl if the cause of the bug is the Next modal?